File: TASK-03/python/subtask3.py
n = int(input("Enter an odd number: "))

# Upper part
for i in range(n // 2 + 1):
    print(" " * (n // 2 - i) + "*" * (2 * i + 1))

# Lower part
for i in range(n // 2 - 1, -1, -1):
    print(" " * (n // 2 - i) + "*" * (2 * i + 1))

File: TASK-03/rust/subtask3.rs
use std::io;

fn main() {
    let mut input = String::new();
    println!("Enter an odd number: ");
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let n: usize = input.trim().parse().expect("Please enter a number");

    // Upper part
    for i in 0..=n / 2 {
        println!("{}{}", " ".repeat(n / 2 - i), "*".repeat(2 * i + 1));
    }

    // Lower part
    for i in (0..n / 2).rev() {
        println!("{}{}", " ".repeat(n / 2 - i), "*".repeat(2 * i + 1));
    }
}
